shared/actions/typed-routes.tsx

```diff
@@ -0,0 +1,14 @@
+// Typed aliases for certain routes so you they are guaranteed to be routed
+// correctly
+
+import * as RouteTreeGen from './route-tree-gen'
```

Comment:
lets not expand on this. i have tickets to make new routing actions and that'll include better typing. this will be deprecated

Reply:
you can keep it for now but i don't expect this file will grow

Reply:
Cool, I think having something like this is very useful to make sure all callers are routing correctly.